src/test/java/com/github/onsdigital/dp/image/api/client/ImageAPIClientTest.java

```diff
@@ -69,17 +64,17 @@ public void testImageAPI_getImages() throws Exception {
         HttpRequestBase httpRequest = captureHttpRequest(mockHttpClient);
 
         // Then no query params are in the URI
-        assertThat(httpRequest.getURI().getQuery()).isNullOrEmpty();
+        assertNull(httpRequest.getURI().getQuery());
```

Comment:
Is this equivalent? The old one was null or empty. I suspect it's fine to be null as the test passes, but just being sure that's what we want.

Reply:
Good point! It is not equivalent but it probably shouldn't be checking both null or empty but be specific about what to expect. I ran it and it is actually a null value, not an empty string.